cognite/client/_api/workflows.py

```diff
@@ -159,18 +159,16 @@ def trigger(
             input (dict | None): The input to the workflow execution. This will be available for tasks that have specified it as an input with the strind "${workflow.input}"
                                 See tip below for more information.
 
-        .. tip::
+        Tip:
             The workflow input can be available in the workflow tasks. For example, if you have a Task with
             function parameters then you can specify it as follows
 
-                >>> from cognite.client.data_classes  import Task, FunctionTaskParameters
-                >>> task = Task(
+                >>> from cognite.client.data_classes  import WorkflowTask, FunctionTaskParameters
```

Comment:
Very nit (2 occurrences):
```suggestion
                >>> from cognite.client.data_classes import WorkflowTask, FunctionTaskParameters
```